test/components/numericAxisTests.ts

```diff
@@ -315,7 +315,7 @@ describe("NumericAxis", () => {
     var yScale = new Plottable.Scale.Linear();
     var yAxis = new Plottable.Axis.Numeric(yScale, "left");
     var yLabel = new Plottable.Component.AxisLabel("LABEL", "left");
-    var barPlot = new Plottable.Plot.VerticalBar(xScale, yScale);
+    var barPlot = new Plottable.Plot.Bar(xScale, yScale);
```

Comment:
Weird that there's a `Plot.Bar` in the `Axis.Numeric` tests. We should probably revisit this to see if we can write the test in a more "unit" way.